cmd/boostd/backup.go

```diff
@@ -291,6 +319,44 @@ var restoreCmd = &cli.Command{
 
 		}
 
+		configFiles, err := ioutil.ReadDir(path.Join(lr.Path(), "config"))
+		if err != nil {
+			return fmt.Errorf("failed to read the restored config files: %w", err)
+		}
+
+		cfgNum := 0
+
+		for _, v := range configFiles {
+			f := v.Name()
+			s := strings.Split(f, ".")
+			if (len(s) == 3) && (s[0] == "config") && (s[1] == "toml") {
+				if ver, err := strconv.Atoi(s[2]); err == nil {
+					if ver > cfgNum {
+						cfgNum = ver
+					}
+				}
+
+			}
+		}
+
+		//Remove default config.toml created with repo. Don't fail if file cannot be removed
+		if err = os.Remove(path.Join(lr.Path(), "config.toml")); err != nil {
+			fmt.Println(fmt.Errorf("failed to read the restored config files: %w", err))
+			fmt.Println("Please create the config.toml link manually")
+		}
+
+		latestCfgFile := "config.toml." + strconv.Itoa(cfgNum)
```

Comment:
@dirkmc I don't like this code. It is too childish. If you have any better suggestion, please let me know.